### PR TITLE
New feature articles

### PR DESCRIPTION
**Pull request by martijnrusschen**

### CODE REVIEW

File: docs/programs/api-tokens.md [hunk 1]
```diff
@@ -7,19 +7,15 @@ id: "programs/api-tokens"
 Your program’s administrative users can generate and manage API tokens to experiment with or use the HackerOne API. The API token identifier and value are used as the username and password for HTTP Basic authentication.
 
 To generate an API token:
-1. Go to <b>Settings > Program > Automation > API</b>.
+1. Go to <b>Organization Settings > API Tokens</b>.
 ![api-token-1](./images/api-token-1.png)
```
Comment (martijnrusschen) [inline docs/programs/api-tokens.md:11]: Need to update screenshots

File: docs/programs/homepage.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Homepage"
+path: "/programs/homepage.html"
+id: "programs/homepage"
+---
+
+The homepage is the central place for customers to go after they log in. This page shows all programs within your organization and any action items required per program. Regardless of the type of program you're running with HackerOne. This page will show a quick overview of the status and the action items of your vulnerability reports.
```
Comment (jessiwright) [inline docs/programs/homepage.md:7]: "Regardless of the ... HackerOne." is a fragment. I suggest 
"This page shows all programs within your organization and any action items required per program, regardless of the type of program you're running."
Or
"Regardless of the type of program you're running with HackerOne, this page will show a quick overview of the status and the action items of your vulnerability reports."

Your intended meaning will determine which is better.

Comment (martijnrusschen) [reply docs/programs/homepage.md]: Fixed it

File: docs/changelog/2021/august.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: "August 2021"
+path: "/changelog/2021/august"
+date: "2021-08"
+---
+
+### Customer Homepage
+The [customer homepage](/programs/homepage.html) is the starting point for customers to see all their programs and activity in their organization. Quickly go to the thing that requires your attention regardless of which program it is in.
```
Comment (karinjvrensburg) [inline docs/changelog/2021/august.md:8]: ... is the starting point for customers to see an overview of all programs and latest report activity and mentions in their organization.

Quickly access the area that requires your attention, regardless of the program it's associated with.

File: docs/changelog/2021/august.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+![homepage](./images/aug_2021_homepage.png)
+
+### Organization-level User Management
+Managing all team members from a central place with our new Organization-level [User Management](/programs/user-management.html). Regardless of which program they can access, you can now manage team members and API access from a single location.
+
+![user-management](./images/aug_2021_users.png)
+
+### Report sidebar
+Improving the user experience while viewing reports. Making it easy to see all relevant information at a glance
```
Comment (karinjvrensburg) [inline docs/changelog/2021/august.md:18]: Improving the user experience while viewing reports. Making it easy to see all relevant information at a glance while taking action on a report

**Pull request merged**